test/creator.ts

```diff
@@ -0,0 +1,30 @@
+import { VNodeChildElement, h } from 'snabbdom/src/package/h';
```

Comment:
Is there a specific reason why not snabbdom-pragma? This package is for testing so size should not really be a concern

Reply:
`snabbdom-pragma` has the same problem as `snabbdom-looks-like`, they are outdated with SnabbDOM 1.x.

Reply:
I see, ok